iothub/device/src/ClientPropertyCollection.cs

```diff
@@ -35,8 +35,7 @@ public void AddRootProperty(string propertyName, object propertyValue)
 
         /// <inheritdoc path="/remarks" cref="AddRootProperty(string, object)" />
         /// <inheritdoc path="/seealso" cref="AddInternal(IDictionary{string, object}, string, bool)" />
-        /// <inheritdoc path="/exception['ArgumentNullException']" cref="AddRootProperty(string, object)" />
-        /// <inheritdoc path="/exception['ArgumentException']" cref="AddRootProperty(string, object)" />
+        /// <inheritdoc path="/exception" cref="AddRootProperty(string, object)" />
```

Comment:
`<inheritdoc path="/exception"` inherits all exceptions listed under the API in `cref`.